var/spack/repos/builtin/packages/callflow/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import stat
+from spack import *
+
+
+class Callflow(PythonPackage):
+    """CallFlow is an interactive visual analysis tool that provides a
+       high-level overview of CCTs together with semantic refinement
+       operations to progressively explore the CCTs."""
+
+    homepage = "https://github.com/LLNL/CallFlow"
+    url      = "https://github.com/LLNL/CallFlow/archive/v1.1.0.tar.gz"
+    git      = 'git@github.com:LLNL/CallFlow.git'
```

Comment:
```suggestion
    git      = 'https://github.com/LLNL/CallFlow.git'
```
This seems to be more reliable if you don't have SSH keys setup.